src/Salt/UserBundle/Resources/views/User/index.html.twig

```diff
@@ -6,23 +6,22 @@
         <form class="form-inline">
             <div class="form-group col-md-4" >
                 <label for="organization">Organization : </label> 
-                <input type ="text"  class="form-control" autocomplete = "off" id="search_form_organization" name="organization">
+                <input type ="text" class="form-control" autocomplete = "off" id="search_form_organization" name="organization">
             </div>
             <div class="form-group col-md-4">
                 <label for="user_role">User Role : </label>
                 <select id ="search_form_user_role" class="form-control">
-                      <option value="">Select</option>
-                      <option value="Super User">Super User</option>
-                      <option value="Super Editor">Super Editor</option>
-                      <option value="Admin">Admin</option>
-                      <option value="Editor">Editor</option>
-                      <option value="User">User</option>
-                  </select>
+                    <option value="">Select</option>
+                    <option value="Super User">Super User</option>
+                    <option value="Super Editor">Super Editor</option>
+                    <option value="Admin">Admin</option>
+                    <option value="Editor">Editor</option>
+                    <option value="User">User</option>
+                </select>
             </div>
         </form>
     </div>
-      <br/><br/>
-
+    <br/>
```

Comment:
Remove all <br> tags. BR tags aren't meant for moving stuff around. Please add a class or utilize and existing class and add margin or padding (whichever is appropriate).